src/NetTopologySuite/Planargraph/DirectedEdge.cs

```diff
@@ -85,7 +85,14 @@ public Edge Edge
         /// Returns 0, 1, 2, or 3, indicating the quadrant in which this DirectedEdge's
         /// orientation lies.
         /// </summary>
-        public int Quadrant => _quadrant;
+        [Obsolete("Use QuadrantValue")]
+        public int Quadrant => (int)_quadrant;
+
+        /// <summary>
+        /// Returns 0, 1, 2, or 3, indicating the quadrant in which this DirectedEdge's
+        /// orientation lies.
+        /// </summary>
+        public Quadrant QuadrantValue => _quadrant;
```

Comment:
Not sure about this name... maybe [`QuadrantEx`](https://stackoverflow.com/q/4039054/1083771)?

Reply:
Should be in sync with how we name it in `EdgeEnd`